packages/netlify-cms-backend-github/src/API.js

```diff
@@ -350,7 +356,7 @@ export default class API {
 
   branchHasPR = async ({ branchName, ...rest }) => {
     const prs = await this.getPRsForBranchName({ branchName, ...rest });
-    return prs.some(pr => this.branchNameFromRef(pr.head.ref) === branchName);
+    return prs.some(pr => pr.head.ref === branchName);
```

Comment:
BUG: Looks like a bug introduced in https://github.com/netlify/netlify-cms/pull/2430

`pr.head.ref` is not the full ref - it is the branch name, so calling `branchNameFromRef` breaks this method

Reply:
Fixed in #2523: https://github.com/netlify/netlify-cms/pull/2523/commits/9a63d11a9be15d0cd8c5a71141774b2175d9b284#diff-b4ba42c688a23ed9cec5139daec2c157R735-R738